src/lexer/Token.ts

```diff
@@ -12,8 +11,6 @@ export interface Token {
     text: string;
     /** True if this token's `text` is a reserved word, otherwise `false`. */
     isReserved: boolean;
-    /** The literal value (using the BRS type system) associated with this token, if any. */
-    literal?: BrsType;
```

Comment:
Removed `literal` property because we're no longer tracking the "real" value, we can derive the Type information from the token kind.